src/components/AttachmentCarousel.js

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
```

Comment:
Can you create a new folder named `AttachmentCarousel` and move this file inside and rename as `index.js.`

Rename `Carousel` component to  `CarouselActions`, also move it inside `AttachmentCarousel` folder.

Reply:
This should be all set!